Update the step object section in adaptive authentication doc (#3779)
wso2/docs-is: en/docs/references/adaptive-authentication-js-api-reference.md

    information.
-   `          step.idp         ` :  Gives the identity provider name which was used
    to authenticate this user.
-   `          step.authenticator         ` :  Gives the authenticator name which was used

>   to authenticate this user. You can find the authenticator names from the table in [here](#core-functions).

Reviewer: ```suggestion
  to authenticate this user. Refer to the table in [core functions section](#core-functions) to find the authenticator names.
```
